app/src/main/java/co/nimblehq/compose/crypto/ui/uimodel/CoinDetailUiModel.kt

```diff
@@ -0,0 +1,34 @@
+package co.nimblehq.compose.crypto.ui.uimodel
+
+import co.nimblehq.compose.crypto.data.extension.orZero
+import co.nimblehq.compose.crypto.domain.model.CoinDetail
+import java.math.BigDecimal
+
+private const val COIN_CURRENCY_KEY = "usd"
```

Comment:
I think it duplicates `HomeViewModel.MY_COINS_CURRENCY`. Should we rename it to something like `FIAT_CURRENCY` for reusing?